### DISCUSSION TITLE
Pharmacy drug type of measurement

### DISCUSSION THREAD

**Discussion by sklukasatphs** [CMSgov/hospital-price-transparency#147] [Q&A] [answered]
VALID VALUE for Milligrams on the slide shows **ME** and not **MG**.  we'd like to double check that is not a miskey.
![image](https://github.com/user-attachments/assets/65b5e8a0-213d-4a05-97a4-4160e839df84)

thanks

**Comment by RidenourK** [comment, marked as answer]
@sklukasatphs. Thank you for your question. 

Please refer to GitHub post #144.
